edx_exams/apps/api/v1/views.py

```diff
@@ -1 +1,184 @@
-# Create your views here.
+"""
+V1 API Views
+"""
+import logging
+import uuid
+
+from edx_api_doc_tools import path_parameter, schema
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from edx_exams.apps.api.permissions import StaffUserPermissions
+from edx_exams.apps.api.serializers import ExamSerializer
+from edx_exams.apps.core.exam_types import get_exam_type
+from edx_exams.apps.core.models import CourseExamConfiguration, Exam
+
+log = logging.getLogger(__name__)
+
+
+class ExamsView(APIView):
```

Comment:
Zach's last comment made me realize this is actually the CourseExamsView - you can't use it to operate on exams outside a course or even a single exam in a course